R/range-selector.R

```diff
@@ -56,7 +56,11 @@ dyRangeSelector <- function(dygraph,
   if (!is.null(dateWindow)) {
     if (length(dateWindow) != 2)
       stop("dateWindow must be vector of length 2 that is convertible to POSIXct")
-    selector$dateWindow <- sapply(USE.NAMES = FALSE, dateWindow, asISO8601Time)
+    # TODO: Rename dateWindow argument?
```

Comment:
I just checked the dygraphs (JS) documentation and they use `dateWindow` also for numeric values (http://dygraphs.com/options.html#dateWindow). Not sure if we should stick with their notation because it may be easier to maintain dygraphs (R) when updating to a new version of dygraphs (JS).